website/docs/r/persistent_volume.html.markdown

```diff
@@ -138,19 +138,19 @@ The following arguments are supported:
 * `azure_disk` - (Optional) Represents an Azure Data Disk mount on the host and bind mount to the pod.
 * `azure_file` - (Optional) Represents an Azure File Service mount on the host and bind mount to the pod.
 * `ceph_fs` - (Optional) Represents a Ceph FS mount on the host that shares a pod's lifetime
-* `cinder` - (Optional) Represents a cinder volume attached and mounted on kubelets host machine. For more info see http://releases.k8s.io/HEAD/examples/mysql-cinder-pd/README.md
```

Comment:
The examples seem to be moved to https://github.com/kubernetes/examples

For example - https://github.com/kubernetes/examples/tree/master/mysql-cinder-pd

Rather than removing these URLs, I think it would be valuable to point to the new URL

Reply:
Apologies if I wasn't clear @neha-viswanathan, I meant that the example links should be fixed throughout your changes, rather than just in this particular case.